library/src/main/java/com/github/clans/fab/FloatingActionMenu.java

```diff
@@ -131,7 +132,10 @@ private void init(Context context, AttributeSet attrs) {
         mLabelsPaddingRight = attr.getDimensionPixelSize(R.styleable.FloatingActionMenu_menu_labels_paddingRight, mLabelsPaddingRight);
         mLabelsPaddingBottom = attr.getDimensionPixelSize(R.styleable.FloatingActionMenu_menu_labels_paddingBottom, mLabelsPaddingBottom);
         mLabelsPaddingLeft = attr.getDimensionPixelSize(R.styleable.FloatingActionMenu_menu_labels_paddingLeft, mLabelsPaddingLeft);
-        mLabelsTextColor = attr.getColor(R.styleable.FloatingActionMenu_menu_labels_textColor, Color.WHITE);
+        mLabelsTextColor = attr.getColorStateList(R.styleable.FloatingActionMenu_menu_labels_textColor);
+        // set default value if null same as for textview
+        if (mLabelsTextColor == null)
+            mLabelsTextColor = ColorStateList.valueOf(0xFF000000);
```

Comment:
Thanks for the contribution!
Could you please change the default color to white and add the curly braces?